newsletter.css

```diff
@@ -266,4 +262,159 @@ footer {
     margin-top: 6.393125em;
     margin-right: 2.5em;
     margin-bottom: 10.0625em;
+} */
```

Comment:
can we remove these comments? if we commit this in then @shravan1k 's code will break. alternatively you can prefix those css tags with `sk-` like you did for yours and let @shravan1k know

Reply:
Sounds good. I made sure Shravan knows!